services/cosi/encode.go

```diff
@@ -0,0 +1,61 @@
+package cosi
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"github.com/dedis/cothority/lib/crypto"
+	"github.com/dedis/cothority/lib/network"
+	"strings"
+)
+
+// MarshalJSON implements golang's JSON marshal interface
+func (s *ServiceResponse) MarshalJSON() ([]byte, error) {
```

Comment:
This is for writing out the received signature. JSON is more human readable (and for instance easier to parse from a web-frontend)